download.js

```diff
@@ -46,7 +46,12 @@ const download = (opt, cb) => {
     }
 
     obj.forEach((entry) => {
-      const pathToLocalFile = path.join(opt.target, entry.key + '.json');
+      let pathToLocalFile = path.join(opt.target, entry.key + (opt.extension || '.json'));
+      // trim the projectId
+      if (pathToLocalFile.indexOf(opt.projectId + '/') > -1) pathToLocalFile = pathToLocalFile.replace(opt.projectId + '/', '');
```

Comment:
as the download is so more optimized for using locally - major is ok...and not expecting to much consumers having issue with it.

having a flag that should be turned on per default and turned off to respect to be no major is not good - semver is all that counts - even if getting to v100.1.1 ;)